modules/user-access/pages/jwt-token.adoc

```diff
@@ -492,3 +492,63 @@ gadmin restart -y
 ====
 Existing environment variable may have some influence on the current use of TigerGraph.
 ====
+
+== Usage of GSQL JWT Token
+Starting from 4.1.0, the token generated by GSQL API is JWT Token and can be used on GSQL and RESTPP server for authentication.
+
+=== create JWT Token
+
+[console]
+----
+POST :14240/gsql/v1/tokens payload '{"secret":"<secret>", "graph":"<graph name>, lifetime: <number in ms>"}'
+----
+
+ * `secret`: the `secret` represents the user who owns so there is no need user login if secret set in the payload
+    if there is no secret specified in the payload a secret will be generated automatically
+                  
+ * `graph`: the `graph` means the scope the token can be used
+    if there is no graph specified the token is global token which is applied to all endpoints
+    
+ * `lifetime`: the `lifetime` means the duration of token can be used and  the time unit is in ms 
```

Comment:
Same as above: The lifetime specifies the duration for which the token can be used, measured in milliseconds. If no lifetime is specified, the default duration is 7 days.